recaptcha_enterprise/snippets/src/test/java/app/SnippetsIT.java

```diff
@@ -63,6 +58,12 @@
 import org.springframework.web.util.UriComponentsBuilder;
 import recaptcha.AnnotateAssessment;
 import recaptcha.GetMetrics;
+import recaptcha.accountdefender.AccountDefenderAssessment;
```

Comment:
You might not have to do this if you use the path `test/java/recaptcha` and package `recaptcha`.